### ISSUE TITLE
Add example to each option and function in api_reference

### ISSUE THREAD

**Issue by jcubic** [jcubic/jquery.terminal-www#22]
It will be easier to just look at the code see what the method/function or option do. This is the same as picure that is like 1000 words.

**Comment by ezinneanne** [comment]
Hello, is this still on?
Can I work on this

**Comment by jcubic** [comment]
Yes, I didn't have time to work on this. If you want to take this go ahead.

**Comment by ezinneanne** [comment]
Hello, some of the documentation's code blocks contain errors in my code editor. I'm using vscode and when I enter some of the code blocks, it marks some as errors and is the same on my browser console.

**Comment by jcubic** [comment]
What errors? Can you show me?

**Comment by ezinneanne** [comment]
Yes, for example when I copied this;
![Screenshot_143](https://user-images.githubusercontent.com/79814820/204474491-4bc4e776-f34c-487b-a6de-3b98489d57e7.png)

When I pasted it in my editor, it was showing this:
![Screenshot_142](https://user-images.githubusercontent.com/79814820/204474661-2da45b3a-4cad-44e7-84d1-54853fd28428.png)


When I checked my browser console, I saw this:
![Screenshot_141](https://user-images.githubusercontent.com/79814820/204474876-9f3ccf61-b38b-41e6-9174-b9a275a5f4d0.png)

**Comment by jcubic** [comment]
it seems that there is a syntax error in the example, it's more like a typo, missing `}` as the error suggests.

If you could fix the errors while you copy that would be great, but you can just copy-paste.
